tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -360,7 +360,11 @@ ipfwd/test_dir_bcast.py:
   skip:
     reason: "Unsupported topology."
     conditions:
-      - "topo_type not in ['t0', 'm0']"
+      - "topo_type not in ['t0', 'm0', 'dualtor']"
+  xfail:
+    reason: "Dualtor do not support now, need to fix in buildimage."
+    conditions:
+      - "topo_type in ['dualtor']"
```

Comment:
Sorry, I have another comment. Can you create a Github issue for dir_bcast on dualtor testbed? Then we can add the github issue as a condition here. Then the xfail mark will be added when the Github issue is still open.

Reply:
OK, I'll add issue link to conditional mark